src/pretalx/schedule/models/room.py

```diff
@@ -23,7 +27,7 @@ class Room(LogMixin, models.Model):
         blank=True,
         verbose_name=_("GUID"),
         help_text=_(
-            "Unique identifier (UUID) to help external tools identify the room."
+            "Unique identifier (UUID) to help external tools identify the room. When left empty, a UUID will be generated automatically from the pretalx instance identifier and local room id."
```

Comment:
That's a very technical way of putting things, which is not really suitable for users (who already end up confused about the difference in naming between "GUID" and "UUID", which is why we've had to hide that part of the form!). Not sure we need to change anything here at all.

Reply:
So as it's already hidden by default, the addition is is okay? Or should I drop the addtional text from this PR?

Reply:
I'd favour dropping the text, as it doesn't add much in understanding,and will force every translation to add the new sentence (and otherwise the whole text will display in English). As we don't have lots of active translators, I'd like to avoid unnecessary modifications to existing strings.